ibm_db2/ibm_db2.php

```diff
@@ -139,7 +139,7 @@ function db2_autocommit ($connection, $value = null) {}
  * from stored procedures. A non-negative value specifies the maximum size of 
  * the XML data that will be retrieved from the database. If this parameter 
  * is not used, a default of 1MB will be assumed for retrieving the XML 
- * output value from the stored procedure.�
```

Comment:
I know this is really strange but I've got a special char with my IDE, this is why I remove this char.
![special_char](https://user-images.githubusercontent.com/2140469/28220771-01f39b84-68c1-11e7-8a6e-4f6da8aed2f4.png)